src/set_theory/game/pgame.lean

```diff
@@ -660,6 +660,102 @@ def relabel_relabelling {x : pgame} {xl' xr'} (el : x.left_moves ≃ xl') (er :
   relabelling x (relabel el er) :=
 relabelling.mk el er (λ i, by simp) (λ j, by simp)
 
+/-- `is_option' x y` means that `x` is either a either a left or right option of `y`, up to
```

Comment:
Is `is_relabelled_option` too long a name? Or is the name not going to appear much anyway?

Reply:
I like that name better, I'll do the rename.